### ISSUE TITLE
requestparser not validating "null" properly

### ISSUE THREAD

**Issue by rishi-freshbooks** [flask-restful/flask-restful#497]
```
post_parser.add_argument(
    'title',
    required=True,
    trim=True
)
```
Does *not* validate / throw a 400 when it encounters this payload:
```
{ "title": null }
```

allowing the request to propagate to a database insert and failing there, since title is not nullable.

Any thoughts?

**Comment by rishi-freshbooks** [comment]
I believe I have the fix here:
https://github.com/flask-restful/flask-restful/pull/498

Please consider!

**Comment by rishi-freshbooks** [comment]
Any update on this? Thanks!

**Comment by jbouzekri** [comment]
+1. This is very blocking because as you said it propagates to the DB if you do not add a manual validation check.

**Comment by joshfriend** [comment]
fixed by #538
